package.json

```diff
@@ -27,7 +28,7 @@
     "precommit": "lint-staged",
     "sassdoc": "sassdoc ./src/patternfly/**/*.scss --parse > sassdoc/data.json",
     "serve": "gatsby serve",
-    "start": "npm run develop",
+    "start": "concurrently \"gulp start\" \"gatsby develop\"",
```

Comment:
ah, folks also use `npm run dev` and `npm run develop` to start the dev server. The main README references `npm run dev` as the way to start the server - https://github.com/patternfly/patternfly-next#development

Can we update those to match? Or maybe consolidate/remove the extras and tell everyone? Also is there a way to replicate `dev:expose` where the server binds to your network IP, and not just localhost/127.0.0.1?